controllers/nginx/pkg/cmd/controller/utils.go

```diff
@@ -39,6 +39,18 @@ func sysctlSomaxconn() int {
 	return maxConns
 }
 
+// sysctlFSFileMax returns the value of fs.file-max, i.e.
+// maximum number of open file descriptors
+func sysctlFSFileMax() int {
+	maxConns, err := sysctl.New().GetSysctl("fs/file-max")
+	if err != nil {
+		glog.Errorf("unexpected error reading system maximum number of open file descriptors (fs.file-max): %v", err)
+		return 0
```

Comment:
Maybe a comment here that this means "don't render", not that the max will be 0 :-)